multiapi-engine/src/test/resources/asyncapigenerator/testFileGeneration/assets/CreateOrderMapper.java

```diff
@@ -1,20 +1,21 @@
-package com.sngular.scsplugin.filegeneration.model.event;
+package com.sngular.scsplugin.filegeneration.model.event.schemas;
 
 import java.util.Objects;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.v3.oas.annotations.media.Schema;
-import com.sngular.scsplugin.filegeneration.model.event.WaiterMapper;
-import com.sngular.scsplugin.filegeneration.model.event.OrderMapper;
 
+@JsonDeserialize(builder = CreateOrderMapper.CreateOrderMapperBuilder.class)
 public class CreateOrderMapper {
 
   @JsonProperty(value ="order")
-  private OrderMapper order;
+  private com.sngular.scsplugin.filegeneration.model.event.schemas.OrderMapper order;
```

Comment:
Why did this change the way we import the classes?

Reply:
The reason is to avoid conflicts when a `message` has the same name than a `schema`; which is the origin of this issue.